tsconfig.json

```diff
@@ -1,11 +1,14 @@
 {
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es2015",
-        "declaration": true,
-        "outDir": "./dist"
-    },
-    "include": [
-        "src/**/*"
-    ]
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
+    "declaration": true,
+    "outDir": "./lib",
+    "strict": true,
+    "jsx": "react",
+    "typeRoots": ["./types", "node_modules/@types"],
```

Comment:
Just copied in the ad-hok types manually for now

Once they're published can remove this